docs/core/windows-prerequisites.md

```diff
@@ -46,18 +46,51 @@ For download links and more information, see [.NET downloads](https://dotnet.mic
 
-## Prerequisites for .NET Core 3.0 Preview 3
+## Prerequisites with Visual Studio
+    
+You can use any editor to develop .NET Core applications using the .NET Core SDK. Visual Studio provides an integrated development environment for .NET Core apps on Windows.
 
-.NET Core 3.0 Preview 3 has the same prerequisites as other versions of .NET Core. However, if you want to use Visual Studio to create .NET Core 3.0 projects, you must use the [Visual Studio 2019](https://visualstudio.microsoft.com/downloads/?utm_medium=microsoft&utm_source=docs.microsoft.com&utm_campaign=inline+link&utm_content=download+vs2019). Visual Studio 2019 can be installed side-by-side with other versions of Visual Studio without conflict.
+<a name="vs-mapping"></a>
 
-## Prerequisites with Visual Studio 2017
-    
-You can use any editor to develop .NET Core applications using the .NET Core SDK. Visual Studio 2017 provides an integrated development environment for .NET Core apps on Windows.
+Each .NET Core version has a minimum version of Visual Studio required. To verify your Visual Studio version:
+
+* On the **Help** menu, choose **About Microsoft Visual Studio**.
+* In the **About Microsoft Visual Studio** dialog, verify the version number.
+
+The following table lists the minimum version for each SDK:
 
-You can read more about the changes in Visual Studio 2017 in the [release notes](/visualstudio/releasenotes/vs2017-relnotes).
+| .NET Core SDK version | Visual Studio version                      |
+| --------------------- | ------------------------------------------ |
+| 3.0                   | Visual Studio 2019 version 16.3 or higher. |
+| 2.2                   | Visual Studio 2017 version 15.9 or higher. |
+| 2.1                   | Visual Studio 2017 version 15.7 or higher. |
+| 1.x                   | Visual Studio 2017 version 15.0 or higher. |
 
 <!-- markdownlint-disable MD025 -->
 
+# [.NET Core 3.0](#tab/netcore30)
+
+To develop .NET Core apps in Visual Studio 2019 using the .NET Core 3.0 SDK:
+
+ 1. [Download and install Visual Studio 2019 version 16.3.0 or higher](/visualstudio/install/install-visual-studio) with the **.NET Core cross-platform development** workload (in the **Other Toolsets** section) selected.
+
+<!-- ![Screenshot of Visual Studio 2019 installation with the ".NET Core cross-platform development" workload selected](./media/windows-prerequisites/vs-2017-workloads.jpg) -->
+
+Visual Studio 2019 16.3 uses .NET Core 3.0 SDK by default after the workload is installed.
+
+If you want your existing projects to use the latest .NET Core runtime, retarget existing .NET Core projects to .NET Core 3.0 using the following instructions:
```

Comment:
Perhaps make it clearer that this is a per-project setting (e.g., retarget each existing .NET Core project...)